index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <title>Aliza Amir Portfolio</title>
+</head>
+<body>
+  <nav>
+    <h3 class="logo">Aliza Amir</h3>
+    <span class="icon">&#9776</span>
```

Comment:
- Kindly make sure that only the hamburger button has a `nav` tag. :+1: 

* The `nav` tag is for semantics reason and the right position for convention and user experience for mobile devices